script/Arcueid/Day 5/05_00_ARC05INTERLUDE.txt

```diff
@@ -71,7 +71,7 @@ But even that salvation was lost long ago.
 [sha:079f100fea4fa140a4b9cbef75777f0dc70ae01c]{
 -- Page 354, Offset 9496.
 -- 「“ァ―――ア、ア―――ォ、ギ―――”」
-"%{i}A――Ah, aa――h, gh%{/i}―――"
+"%{i}Uurgh――uuah――aah――ugh――aakhhhh%{/i}..."
```

Comment:
Hmm, dashes feels a bit too much when he says it kinda fast and in a rather consistent string...